charts/operator/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: intel-device-plugins-operator
+description: A Helm chart for Intel Device Plugins Operator for Kubernetes
+type: application
+version: 0.1.0
+appVersion: "0.23.0"
```

Comment:
we have `make set-version` helper that helps to manage the release versions in various places. Can you check the Helm related components get updated properly too?